docs/articles_en/openvino-workflow/model-optimization-guide/weight-compression.rst

```diff
@@ -354,7 +354,7 @@ To find the optimal weight compression parameters for a particular model, refer
 `example <https://github.com/openvinotoolkit/nncf/tree/develop/examples/llm_compression/openvino/tiny_llama_find_hyperparams>`__ ,
 where weight compression parameters are being searched from the subset of values.
 To speed up the search, a self-designed validation pipeline called
-`WhoWhatBench <https://github.com/openvinotoolkit/openvino.genai/tree/master/llm_bench/python/who_what_benchmark>`__
+`WhoWhatBench <https://github.com/openvinotoolkit/openvino.genai/tree/master/tools/who_what_benchmark>`__
```

Comment:
should we point to releases/2024/6 ? and below as well

Reply:
For 2024.6, we will find all references to 'master' and change them to 24.6 (because this version will remain even after 2025.0).
I'm not sure which approach will be better in the long run, though. Aligning versions with every release may be difficult to maintain.

Reply:
> I'm not sure which approach will be better in the long run, though. Aligning versions with every release may be difficult to maintain.

but you have to, because links are updated with new information which is not relevant to older releases

on master branches we can also update file or move it, so the links are invalid as well.